Content.Server/StationRecords/Systems/GeneralStationRecordConsoleSystem.cs

```diff
@@ -30,6 +42,52 @@ private void UpdateUserInterface<T>(EntityUid uid, GeneralStationRecordConsoleCo
         UpdateUserInterface(uid, component);
     }
 
+    private void OnButtonPressed(EntityUid uid, GeneralStationRecordConsoleComponent component,
```

Comment:
ditto, you need to make this a separate system for this instead of expanding the system that deals with general station records (alternatively, you can just rename this entire thing to security records if you intend on making general station records inaccessible)